### PR TITLE
Question Page UI fixes

### PR DESCRIPTION
**Pull request by theresa-lee**
- Removed the question stepper
- Removed the question set selector
- Added url parameter 'qid' to get the question from the menu page
- Removed any old code that had to do with switching the question/question indexes

### CODE REVIEW

File: client/menu/components/MenuQuestionCardDirective.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ tieMenu.directive('menuQuestionCard', [function() {
     },
     template: `
       <div class="tie-menu-question">
-        <a href="../question/question.html">
+        <a href="../question/question.html?qid={{questionId}}">
```
Comment (seanlip) [inline client/menu/components/MenuQuestionCardDirective.js]: ng-href

File: client/menu/components/MenuQuestionCardDirective.js [hunk 2]
```diff
@@ -67,8 +67,8 @@ tieMenu.directive('menuQuestionCard', [function() {
         }
       </style>
     `,
-    controller: ['$scope', 'QuestionDataService',
-      function($scope, QuestionDataService) {
+    controller: ['$scope', '$location', 'QuestionDataService',
```
Comment (seanlip) [inline client/menu/components/MenuQuestionCardDirective.js]: $location added but not used?

File: client/question/components/LearnerViewDirective.js [hunk 3]
```diff
@@ -884,15 +812,16 @@ tie.directive('learnerView', [function() {
...
-          $scope.currentQuestionIndex = 0;
-          $scope.questionIds = $scope.questionSet.getQuestionIds();
-          $scope.questionsCompletionStatus = [];
-          $scope.loadingIndicatorIsShown = false;
-          for (var idx = 0; idx < $scope.questionIds.length; idx++) {
-            $scope.questionsCompletionStatus.push(false);
-          }
           $scope.autosaveTextIsDisplayed = false;
-          loadQuestion($scope.questionSet.getFirstQuestionId());
+          // If there isn't a specificed qid, use the default. If there is one,
```
Comment (seanlip) [inline client/question/components/LearnerViewDirective.js]: typo: specified

File: client/question/question.js [hunk 4]
```diff
@@ -19,6 +19,16 @@
 window.tie = angular.module('tie', ['ui.codemirror', 'tieConfig', 'tieData']);
 
 /**
+ * Setting app to html5mode so that we don't have to have #'s in the url.
```
Comment (seanlip) [inline client/question/question.js]: Shouldn't this go in the global tie config for all pages?

File: client/question/question.js [hunk 5]
```diff
@@ -562,3 +572,11 @@ tie.constant('DEFAULT_AUTOSAVE_SECONDS', 5);
  * @constant
  */
 tie.constant('DISPLAY_AUTOSAVE_TEXT_SECONDS', 1);
+/**
+ * Question Id of the question in case there isn't a question id
```
Comment (seanlip) [inline client/question/question.js]: Default question ID to use if no qid parameter is specified in the URL.

File: client/menu/components/MenuQuestionCardDirective.js (hunk 1 above)
Comment (theresa-lee) [reply client/menu/components/MenuQuestionCardDirective.js]: Done.

File: client/menu/components/MenuQuestionCardDirective.js (hunk 2 above)
Comment (theresa-lee) [reply client/menu/components/MenuQuestionCardDirective.js]: Removed.

File: client/question/components/LearnerViewDirective.js (hunk 3 above)
Comment (theresa-lee) [reply client/question/components/LearnerViewDirective.js]: Done.

File: client/question/question.js (hunk 5 above)
Comment (theresa-lee) [reply client/question/question.js]: Done.

File: client/question/question.js (hunk 4 above)
Comment (theresa-lee) [reply client/question/question.js]: Mm not sure why, but when I tried it in the config file, it wouldn't route correctly to the questions.

Comment (theresa-lee) [reply client/question/question.js]: I'm silly, that's why. Moved.

**Pull request merged**